libbeat/docs/upgrading.asciidoc

```diff
@@ -1,160 +1,329 @@
+process. {beats} typically maintain backwards compatibility for configuration
+settings and exported fields. Please review the
+<<release-notes,release notes>> for potential exceptions.
 
-Upgrading between non-consecutive major versions (e.g. 1.x to 6.x) is not
-supported. If you are on version 1.x of the Beats, we recommend first upgrading
-to 5.6.
+Upgrading between non-consecutive major versions (e.g. 5.x to 7.x) is not
+supported.
 
-[[upgrading-5-to-6]]
-=== Upgrade from 5.x to 6.x
+[[upgrading-6-to-7]]
+=== Upgrade from 6.x to 7.x
 
-Before upgrading your Beats, please review the <<breaking-changes, breaking
-changes>> doc.
+Before upgrading your {beats}, review the <<breaking-changes,breaking changes>>.
 
-// TODO: link to the stack upgrade docs
-//
-// If you are planning an upgrade of the full stack (Elasticsearch, Kibana, or
-// Logstash, in addition to Beats), please read the stack upgrade guide.
+If you're upgrading other products in the stack, also read the
+{stack-ref}/index.html[Elastic Stack Installation and Upgrade Guide]. 
+
+We recommend that you fully upgrade {es} and {kib} to version 7.0
+before upgrading {beats}. If you're on {beats} 6.0 through 6.6,
+upgrade to {beats} 6.7 *before* doing the {es} upgrade.
 
-We recommend that you fully upgrade Elasticsearch and Kibana to version 6.0
-before upgrading Beats. If you are on a Beats 5.x version lower than 5.6, please
-follow the <<upgrading-to-5.6>> section *before* doing the Elasticsearch
-upgrade.
+Upgrading between non-consecutive major versions (e.g. 5.x to 7.x) is not
+supported.
 
-If you're not ready to upgrade Elasticsearch and Kibana to 6.0, that's alright.
-Beats version 6.0 works with Elasticsearch and Kibana version 5.6, so you can
-upgrade Beats now and the rest of the stack later.
+IMPORTANT: Please read through all upgrade steps before proceeding. These steps
+are required before running the software for the first time.
 
-[[upgrading-to-5.6]]
-==== Upgrade to 5.6
+// TODO: We should add step-by-step instructions to tell users to back up the
+// registry, copy over the config file, etc. Similar to what Kibana does:
+// https://www.elastic.co/guide/en/kibana/master/upgrade-standard.html
 
-The upgrade procedure assumes that you have Beats version 5.6 installed. If you
-are on a previous 5.x version of Beats, please upgrade to version 5.6 first.
-The reason is that the Elasticsearch mapping template in 5.6 was modified to be
-compatible with Elasticsearch 6.0 (by removing the `_all` settings).
+[[upgrading-to-6.7]]
+==== Upgrade to {beats} 6.7 before upgrading to 7.0
 
-For this reason, after upgrading to 5.6, you need to make sure that the 5.6
-template is loaded. You can do this by temporarily enabling the
-`output.elasticsearch.template.overwrite` setting, for example with Metricbeat:
+The upgrade procedure assumes that you have {beats} 6.7 installed. If you're on
+a previous 6.x version of {beats}, upgrade to version 6.7 first. This is
```

Comment:
In the same vein as my comment above, should we another sentence here:

> If you're on a previous 6.x version of Beats, upgrade to version 6.7 first. If you're using other products within the Elastic Stack, upgrade Beats as part of the [Elastic Stack upgrade process](#). Upgrading to 6.7 is required because...